dist/angular-block-ui.css

```diff
@@ -81,15 +81,13 @@ body.block-ui > .block-ui-container,
 /* - - - - - - 8-< - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - */
 
 .block-ui-anim-fade > .block-ui-container {
-  -webkit-transition: height 0s linear 200ms, opacity 200ms ease 0s;
-          transition: height 0s linear 200ms, opacity 200ms ease 0s;
+  transition: height 0s linear 200ms, opacity 200ms ease 0s;
 }
```

Comment:
I don't think the CSS transitions have anything to do with the restore bug in IE